Formula/orientdb.rb

```diff
@@ -48,6 +43,16 @@ def post_install
     (var/"log/orientdb").mkpath
     touch "#{var}/log/orientdb/orientdb.err"
     touch "#{var}/log/orientdb/orientdb.log"
+
+    # set the server's root user  password to changeme
+    ENV["ORIENTDB_ROOT_PASSWORD"] = "changeme"
```

Comment:
The problem is that we don't have a default value, the root user password is autogenerated by the db server at first startup and it is stored in hash form. We can change the value to orientdb or something else.